verilog/tools/syntax/README.md

```diff
@@ -188,6 +193,45 @@ directly to a named grammar construct in the [SV-LRM]. The
 [`verilog/CST`](../../CST) library provides functions that abstract away
 internal structure.
 
+## JSON output description
+
+JSON root is an object which maps each input file name to an object containing
+parsing result for that file.
+
+### Parsing result object
+
+| Key      | Type   | Description                                              |
+|----------|--------|----------------------------------------------------------|
+| `tree`   | object | Parser tree. Present only when `--printtree` flag is specified and there were no parsing errors. |
```

Comment:
FYI, it is possible for a file with syntax errors to have a partial syntax tree due to error-recovery, but that detail may not be necessary.  If there are recovered syntax errors, they will still be available in the original errors list.

Reply:
Changed to "Present only when `--printtree` flag is specified and parsing errors didn't prevent tree creation."